app/src/main/java/com/dhbw/brainstorm/MainActivity.kt

```diff
@@ -2,10 +2,104 @@ package com.dhbw.brainstorm
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.JsonReader
+import android.view.View
+import android.widget.Toast
+import com.dhbw.brainstorm.api.CommonClient
+import com.google.gson.Gson
+import kotlinx.android.synthetic.main.activity_main.*
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.converter.scalars.ScalarsConverterFactory
+import java.util.concurrent.TimeUnit
 
 class MainActivity : AppCompatActivity() {
+    val BACKEND_URL = "https://brainstorm-dhbw-backend.herokuapp.com"
```

Comment:
evtl als in Strings.xml packen, oder was isn da best Practice

Reply:
Aye, da haste Recht. Mach ich noch schnell